storage/objectstorage/fake/fake_client.go

```diff
@@ -0,0 +1,34 @@
+package fake
```

Comment:
for visibility please implement this fake client, directly in your _test.go file
Don't need to have a specific file/package used only by your test